### ISSUE TITLE
Enchanting materials on Auction House.

### ISSUE THREAD

**Issue by knasen88** [LightsHope/server#916]
This is not a BUG per se, but it is a huge issue for this server.
There are currently 44 pages of Large Brilliant Shards on horde auctionhouse. But only about 250 "average" priced ones, meanwhile there are 939 and 750 posted by the same user in single auctions as to scam people without an auction addon or just lazy to scan all the pages to find the "real" price.
Example given:
https://streamable.com/vuqcz
This is true for all versions of high level enchanting mats.
Solution: add deposit fee to enchanting materials, because them able to keep them up indefinetely without any risk of loosing a deposit fee is what creates this problem.

2200 shards and 75%~ is two users jewing the system wasting everyones time and scamming players unaware of this. Scamming is against the TOS and this is imo, high level scamming.

below this line you will find YOUR stance on people scamming players.
---------------------------------------------------------------------------------------------------
B. Mute or temporary account interruption

This sanction can last anywhere from 3 hours to 14 days, depending on severity, and is given for repeated violation of the rules, or for any of the following:

    Scamming;

    Lying to, or attempting to deceive Light's Hope staff;

    Abuse of in-game bugs;

    Organized killing for farming rank and/or honor;

    Same faction / PvE server griefing on world bosses;

    Tagging a world boss and having the other faction kill it for you.

C. Permanent ban

The most serious sanction. This sanction is applied in case of multiple violations of the rules, or after multiple account interruptions, as well as any of the following:

    COD Scamming;

    Hacking;

    Using any means of botting (e.g. a program);

    Any client modification (libraries, executables, MPQ, etc.);

    Revealing the identity of an Light's Hope staff member;

    Advertising for other private servers or gold selling websites;

    Selling/Buying player names;

    Exchanging accounts, gold, items, or any ingame services for another game's features;

    Selling/Buying accounts, gold, items, or any ingame services with real money;

    Possession of any item(s) or currencies obtained with illegal methods;

    Publishing software, methods or articles on cheating which can be applied on the server;

    Abuse of major in-game bugs;

    Staff impersonation.

**Comment by Omarthegreatone** [comment]
I agree this is a huge issue when trying to get some decent priced shards for silithus farming

**Comment by Cabro** [comment]
Not Blizzlike though.

**Comment by knasen88** [comment]
Fuck blizzlike, This server has done multiple non blizzlike decisions for the better off it.

**Comment by fgsfdss** [comment]
Make it a bannable offence to post more than X amount of an auction instead of adjusting code.

**Comment by knasen88** [comment]
https://i.imgur.com/hJFigKl.png
can u do something please
